scripts/verify-turnin

```diff
@@ -32,105 +32,123 @@
 #                                                                        #
 ##########################################################################
 
-import sys,os
-import fnmatch, re, getpass
+import fnmatch
+import getpass
+import os
+import re
+import sys
+
+ver = "1.1"
```

Comment:
Since the programs have a common `man` page, etc. it's probably okay to sync the two versions, of the `turnin` and `verify-turnin`.

Reply:
You are suggesting that both `turnin` and `verify-turnin` have identical versions? In this case **3.14**?

Reply:
Yes, they should both be version `3.1.4`.

Reply:
If you are going to do this, please make `verify-turnin` report the `turnin` version so that we don't have to keep the version up to date in more than one places. You can do the same for the copyright notices etc...

i.e., make `verify-turnin -V` run `turnin -V`